docs/remote_firecracker_snapshots.md

```diff
@@ -0,0 +1,82 @@
+# Remote Firecracker snapshots
```

Comment:
please see my comments in the other file. let's just merge the two docs and structure them well.

Reply:
Fixed.